.github/CODEOWNERS

```diff
@@ -158,6 +164,7 @@
 
 # tools
 # TODO: /tools
+tools/lldb-moduleimport-test/                       @adrian-prantl
 /tools/SourceKit                                    @ahoppen @bnbarham @rintaro
 /tools/swift-ide-test                               @ahoppen @bnbarham @rintaro
```

Comment:
Sorry to be nitpicky.

* The order is ASCII, so lowercase letters are ordered before uppercase ones.
* Could we be consistent with the leading `/`?